### DISCUSSION TITLE
Tamanho do terminal

### DISCUSSION THREAD

**Discussion by sulevansampaio** [IBM/tnz#76] [Q&A]
Estou usando tnz.py3270 e estou com dificuldade para retirar o autosize e fixar em 24x80

**Comment by Christopher-Chappell** [comment]
You should be able to put `SET SESSION_PS_SIZE 24x80` either as a command or in your ~/.ztirc file

**Reply by v1gnesh** [reply]
@Christopher-Chappell, are you able to share some sample parms that can be set in `.ztirc`.
Maybe I haven't looked patiently enough, but the doco doesn't detail it...?
